deployment/molecule/ui/tests/test_ui.py

```diff
@@ -0,0 +1,32 @@
+import os
+import pytest
```

Comment:
which test here does check that the UI is running - the server responds with a correct content?

Reply:
I've added two tests:
- Check that the server responds with html with proper title
- Checks that returned html does not contain `error` string (with case insensitivity)

Reply:
thanks!